modules/rest-api/pages/rest-set-up-services-existing-nodes.adoc

```diff
@@ -0,0 +1,140 @@
+
+NOTE: The rebalance occurs even if no `<service>` is changed.
+
+Each assigned service is allocated a default amount of memory.
+You can modify these allocations as needed.
+
+For information about allocating memory per service, see xref:rest-api:rest-configure-memory.adoc[Configuring Memory].
+
+To fetch the details of current memory allocation for each service, see xref:rest-api:rest-get-memory-information.adoc[Getting Memory Information].
+
+== Required Permissions
+
+`Full Admin` role.
+
+[#responses]
+== Responses
+
+|===
+|Value | Description  
+
+| `200 OK` and JSON array containing the expired backups and their details.
+| Successful call.
+
+| `400`
+| Invalid parameter.
+
+| `400 Object Not found`
+| The repository in the endpoint URI does not exist.
+
+| `401 Unauthorized`
+|  Authorization failure due to incorrect username or password.
+
+| `403 Forbidden`, plus a JSON message explaining the minimum permissions.
+| The provided username has insufficient privileges to call this method.
+
+| `404 Object Not Found`
+| Error in the URI path.
+
+| `500 Internal Server Error`
+| Error in Couchbase Server.
+|===
+
+=== Example
+
+For illustration, consider 5 nodes in the cluster with the following services:
+
+* 172.23.108.69 – Data, Index, Query
+* 172.23.108.70 – Data, Search, Query
+* 172.23.108.71 – Query
+* 172.23.108.72 – Backup
+* 172.23.108.73 – Analytics
+
+Reconfigure the services on the existing nodes so that the following is the required output of services on each of the 5 nodes:
+
+* 172.23.108.69 – Data, Query
+* 172.23.108.70 – Data, Index
+* 172.23.108.71 – Index, Query
+* 172.23.108.72 – Search, Query
+* 172.23.108.73 – Analytics
+
+Run the following command to reconfigure the services on the existing nodes and get the required output.
+
+----
+curl -u Administrator:password -X POST 172.23.108.69:8091/controller/rebalance \
+-d "knownNodes=ns_1@172.23.108.69,ns_1@172.23.108.70,ns_1@172.23.108.71,ns_1@172.23.108.72,ns_1@172.23.108.73"  \
+-d "topology[index]=ns_1@172.23.108.70,ns_1@172.23.108.71"  \
+-d "topology[fts]=ns_1@172.23.108.72"  \
+-d "topology[n1ql]=ns_1@172.23.108.69,ns_1@172.23.108.71,ns_1@172.23.108.72" \
+-d "topology[cbas]=ns_1@172.23.108.73"
+---- 
+
+== See Also
+
+* To retrieve the list of services running on a node, see xref:rest-api:rest-list-node-services.adoc[Listing Node Services].
+
+* For information about allocating memory per service, see xref:rest-api:rest-configure-memory.adoc[Configuring Memory].
+
+* To fetch the details of current memory allocation for each service, see xref:rest-api:rest-get-memory-information.adoc[Getting Memory Information].
+
+* For information about assigning services to existing nodes from the UI and CLI, see xref:manage:manage-nodes/modify-services-and-rebalance.adoc[Modify Services and Rebalance].
+
+* For information about assigning services to a new node, see xref:rest-api:rest-set-up-services.adoc[Assigning Services to a New Node].
```

Comment:
`* For information about assigning services to a new node, see xref:rest-api:rest-set-up-services.adoc[Assigning Services to a New Single Node].`